seeding/create-buildings.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# If you're on Windows, run bash create-buildings.sh -w
+# Otherwise, run bash create-buildings.sh
+
+# Import common functions
+SEEDING_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source $SEEDING_DIR/seed.sh
+
+# Run SQL script in docker container
+run_sql_script "
+DELETE FROM buildings;
+INSERT INTO buildings (
+  id,
+  address,
+  name
+)
+VALUES (
+  1,
+  '144 Erb St. W',
+  'Building 1'
```

Comment:
`name` in our case is just an abbreviation of the `address` - so here, it would be `144`